source/_components/binary_sensor.flic.markdown

```diff
@@ -30,13 +30,18 @@ Configuration variables:
 - **host** (*Optional*): The IP or hostname of the flic service server. (default: `localhost`)
 - **port** (*Optional*): The port of the flic service. (default: `5551`)
 - **discovery** (*Optional*): If `true`, the component is configured to constantly scan for new buttons. (default: `true`)
+- **timeout** (*Optional*): Maximum time in seconds an event can be queued locally on a button before discarding the event. (default: `3`)
 
 
 #### {% linkable_title Discovery %}
 
 If discovery is enabled, you can add a new button by pressing it for at least 7s. The button will be paired with the flic service and added to Home Assistant. Otherwise, you have to manually pair it with the flic service. The Home Assistant platform will not scan for new buttons and will only connect to buttons already paired.
 
 
+### {% linkable_title Timeout %}
+When the flic button is triggered while disconnected from flicd, it will queue all events and try to connect and transmit them as soon as possible. The timeout variable can be used to stop events from triggering if too much time passed between the action and the notification in Home Assistant.
```

Comment:
*Flic service* is used for *flicd* in the other parts of the page. This should be consistent by using either *flicd* or *flic service* in the whole documentation.